From 4877d29ad1858671d5a7edcdb1ffed2131d49a58 Mon Sep 17 00:00:00 2001
From: Stefan Lellwitz <lel@iesy.com>
Date: Tue, 19 Jul 2022 15:04:24 +0200
Subject: [PATCH 1/1] arm64: dts: iesy: px30: set drive strength for i2c0 to
 8mA

This resolves i2c bus conflict. Console messages won't be destroyed anymore when using mikroBUS modules.

Signed-off-by: Stefan Lellwitz <lel@iesy.com>
---
 arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi.dts | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi.dts b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi.dts
index 1852d323ffec..d5c708453290 100644
--- a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi.dts
+++ b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi.dts
@@ -999,13 +999,21 @@
 		};
 	};
 
+	i2c0 {
+		i2c0_xfer: i2c0-xfer {
+			rockchip,pins =
+				<0 RK_PB0 1 &pcfg_pull_none_8ma>,
+				<0 RK_PB1 1 &pcfg_pull_none_8ma>;
+		};
+	};
+
 	i2c1 {
 		i2c1_xfer: i2c1-xfer {
 			rockchip,pins =
 				<0 RK_PC2 1 &pcfg_pull_none_8ma>,
 				<0 RK_PC3 1 &pcfg_pull_none_8ma>;
-			};
 		};
+	};
 
 	lt8912 {
 		pinctrl_lt8912: pinctrl-lt8912 {
-- 
2.30.2

